docker-compose.yml

```diff
@@ -30,7 +30,8 @@ services:
       # setup_configuration env vars
       - OBJECTS_DOMAIN=web:8000
       - OBJECTS_ORGANIZATION=Objects
-      - OBJECTTYPES_API_ROOT=https://objecttypes.example.com/api/v2/
+      - OBJECTS_OBJECTTYPES_CONFIG_ENABLE=false
+      # - OBJECTTYPES_API_ROOT=https://objecttypes.example.com/api/v2/
```

Comment:
Is it intentional to disable this?

Reply:
Yes it's a temporary fix for quick-start action
`OBJECTTYPES_API_ROOT` setting was incorrect (it's fixed in this PR), but quick-start doesn't use the current build, but only the published one. 
So to make it work for now `OBJECTTYPES_API_ROOT` should be empty